GameData/BoringCrewServices/Patches/ConnectedLivingSpaces.cfg

```diff
@@ -7,7 +7,7 @@
 	}
 }
 
-@part[BCS_Centauri_CrewCapsule]:NEEDS[ConnectedLivingSpace]:FOR[BoringCrewServices]
+@part[BCS_Centauri_CrewCapsule|BCS_Centauri_MainChute|BCS_Centauri_DrogueChutes]:NEEDS[ConnectedLivingSpace]:FOR[BoringCrewServices]
```

Comment:
This feels unnececary per the same reason as discussed at the start [here](https://github.com/zorg2044/BoringCrewServices/issues/5#issuecomment-2409099311)

Reply:
Closed the request